.circleci/config.yml

```diff
@@ -221,28 +233,54 @@ jobs:
 
   # defines end 2 end tests environment
   # Will spin up an appropriate environment and run the tests against it
+  # Needs AWS authentication + kubectl tool
+  # Attaches persisted CA + certs + keys in the workspace for Auth tests
+  # Uses kubectl port forward to open up the remote qkm on the given k8s cluster + namespace
   e2e:
     machine:
       image: ubuntu-2004:202010-01
-    resource_class: xlarge
+    resource_class: large
+    parameters:
+      test_data:
+        type: string
+        default: $QA_TEST_DATA
+      cluster-name:
+        type: string
+        default: "orchestrate"
+      namespace:
+        type: string
+        default: "qa-qkm"
+      port-frowarding:
+        default: $PORT_FORWARDING
+        type: string
     steps:
       - run:
           name: Uninstall Go
           command: sudo rm -rvf /usr/local/go/
       - go/install:
           version: 1.16.8
       - prepare_golang
+      - prepare_test_data:
+          test_data: << parameters.test_data >>
+      - kubernetes/install
+      - aws-eks/update-kubeconfig-with-authenticator:
+          cluster-name: << parameters.cluster-name >>
+      - when:
+          condition:
+            equal: [ "true", "${<< parameters.port-frowarding >>}" ]
+          steps:
+            - run:
+                name: Port forward
+                command: |
+                  kubectl port-forward -n << parameters.namespace >> service/quorum-key-manager-quorumkeymanager 8080:8080 8081:8081&
```

Comment:
Please use background task instead of this
```
      - run:
          name: Background the app
          background: true
          command: |
```

Reply:
Ok, I didn't know this one